### PR TITLE
Kivy test case : Msg sending and testing sent box updated

### PR DESCRIPTION
**Pull request by shekhar-cis**
Created 2 methods, one for sending message to a static address, another is to check the sent box is updated.

### CODE REVIEW

File: src/bitmessagekivy/tests/test_sent_message.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        # Checking State of Sender's Address Input Field (Empty)
+        self.assertExists('//DropDownWidget/ScrollView[0]//MDTextField[@text=\"\"]', timeout=3)
+        # Checking State of Receiver's Address Input Field (Empty)
+        self.assertExists('//DropDownWidget/ScrollView[0]//MyTextInput[@text=\"\"]', timeout=2)
+        # Checking State of Subject Input Field (Empty)
+        self.assertExists('//DropDownWidget/ScrollView[0]//MyMDTextField[@text=\"\"]', timeout=2)
+        # Click on Send Icon to check validation working
+        self.cli.wait_click('//MDActionTopAppBarButton[@icon=\"send\"]', timeout=2)
+        # Checking validation Pop up is Opened
+        self.assertExists('//MDDialog[@text=\"Please fill the form completely\"]', timeout=2)
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_sent_message.py:35]: make the text into class variable

File: src/bitmessagekivy/tests/test_sent_message.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        # Click on 'Ok' to dismiss the Popup
+        self.cli.wait_click('//MDFlatButton[0]', timeout=2)
+        # Checking Pop up is closed
+        self.assertExists("//ScreenManager[@current=\"create\"]", timeout=2)
+        # ADD SENDER'S ADDRESS
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_sent_message.py:40]: start a new method here

File: src/bitmessagekivy/tests/test_sent_message.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        self.assertExists('//DropDownWidget/ScrollView[0]//ScrollView[0]/MDTextField[@text=\"\"]', timeout=2)
+        # ADD BODY
+        self.cli.setattr(
+            '//DropDownWidget/ScrollView[0]/BoxLayout[0]/ScrollView[0]/MDTextField[0]', 'text', self.test_body)
+        # Checking BODY is Entered
+        self.assertNotEqual('//DropDownWidget/ScrollView[0]/BoxLayout[0]/ScrollView[0]/MDTextField[text]', '')
+        # click on send icon
+        self.cli.wait_click('//MDActionTopAppBarButton[@icon=\"send\"]', timeout=2)
+        # Checking validation so Pop up is Opened
+        self.assertExists('//MDDialog[@text=\"Please fill the form completely\"]', timeout=2)
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_sent_message.py:69]: also variable

File: src/bitmessagekivy/tests/test_sent_message.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        # clicked on ok button to close popup
+        self.cli.wait_click('//MDFlatButton[0]', timeout=2)
+        self.assertExists("//ScreenManager[@current=\"create\"]", timeout=5)
+        # ADD RECEIVER ADDRESS
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_sent_message.py:73]: merge from here with the method after this, also change names to reflect

File: src/bitmessagekivy/tests/test_sent_message.py [hunk 5]
```diff
@@ -0,0 +1,101 @@
+from .telenium_process import TeleniumTestProcess
+from .common import skip_screen_checks
+from .common import ordered
+
+test_address = {'receiver': 'BM-2cWmjntZ47WKEUtocrdvs19y5CivpKoi1h'}
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_sent_message.py:5]: here try the autoresponder

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/bitmessagekivy/tests/test_sent_message.py (hunk 1 above)
Comment (shekhar-cis) [reply src/bitmessagekivy/tests/test_sent_message.py]: Resolved

File: src/bitmessagekivy/tests/test_sent_message.py (hunk 5 above)
Comment (shekhar-cis) [reply src/bitmessagekivy/tests/test_sent_message.py:5]: Added autoresponder_address as receiver address

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**